ocaml/database/schema.ml

```diff
@@ -82,7 +82,11 @@ module Table = struct
     columns: Column.t list;
     persistent: bool;
   } [@@deriving sexp]
-  let find name t = List.find (fun col -> col.Column.name = name) t.columns
+  let find name t =
+    try
+      List.find (fun col -> col.Column.name = name) t.columns
+    with Not_found ->
```

Comment:
Thanks it is always annoying when you get `Not_found` in the logs not knowing what was not found...

Reply:
Indeed!